cdap-docs/developer-manual/source/metadata/field-lineage.rst

```diff
@@ -0,0 +1,195 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2018 Cask Data, Inc.
+
+===================
+Field Level Lineage
+===================
+
+
+.. _metadata-fieldlevellineage:
+
+Introduction
+============
+CDAP provides a way to retrieve the lineage for dataset entities. Dataset can have
+schema associated with it. Schema defines different fields in the dataset and their
```

Comment:
Do you mean...

A dataset entity can have associated schema. The scheme defines the different fields in the dataset and their respective information type.

Reply:
Changed to `The schema defines different fields in the dataset along with their data type information.` 

Dataset has fields and each field has name and data type for example String, Int etc.